### DISCUSSION TITLE
Default Options

### DISCUSSION THREAD

**Discussion by jxnu-liguobin** [kobylynskyi/graphql-java-codegen#442] [General]
The default value has been set in the plugin. At this time, it is always false here (if statement) 
```java
       if (mappingConfig.getGenerateBuilder() == null) {
            mappingConfig.setGenerateBuilder(false);
        }
        if (mappingConfig.getGenerateImmutableModels() == null) {
            mappingConfig.setGenerateImmutableModels(true);
        }
```
The default values in the library do not take effect and are useless.
Modify the default value of the plugin or?

**Comment by kobylynskyi** [comment]
@jxnu-liguobin, just wondering: why would someone want to override a default value of the config?

**Reply by jxnu-liguobin** [reply]
The current default value is not work.
